CONTRIBUTING/index.html

```diff
@@ -1,4168 +1,10 @@
-
-<!doctype html>
-<html lang="en" class="no-js">
+<html xmlns="http://www.w3.org/1999/xhtml">
   <head>
-    
-      <meta charset="utf-8">
-      <meta name="viewport" content="width=device-width,initial-scale=1">
-      
-      
-      
-      <link rel="icon" href="../assets/images/favicon.png">
-      <meta name="generator" content="mkdocs-1.2.4, mkdocs-material-8.1.9">
-    
-    
-      
-        <title>Contributing - Argo Workflows - The workflow engine for Kubernetes</title>
-      
-    
-    
-      <link rel="stylesheet" href="../assets/stylesheets/main.2b4465f4.min.css">
-      
-        
-        <link rel="stylesheet" href="../assets/stylesheets/palette.e6a45f82.min.css">
-        
-      
-    
-    
-    
-      
-        
-        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&display=fallback">
-        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
-      
-    
-    
-    <script>__md_scope=new URL("..",location),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
-    
-      
-  
-
-
-  
-  
-
-
-  <script>function gtag(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","G-5Z1VTPDL73"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){this.value&&gtag("event","search",{search_term:this.value})}),"undefined"!=typeof location$&&location$.subscribe(function(e){gtag("config","G-5Z1VTPDL73",{page_path:e.pathname})})})</script>
-  <script async src="https://www.googletagmanager.com/gtag/js?id=G-5Z1VTPDL73"></script>
-
-
-    
-    
+    <title>Contributing - Argo Workflows - The workflow engine for Kubernetes</title>
+    <meta http-equiv="refresh" content="0;URL=https://argo-workflows.readthedocs.io/en/latest/CONTRIBUTING/'" />
```

Comment:
shouldn't these all link to `stable` same as #12360?

also we'll want to add a [canonical reference](https://developers.google.com/search/docs/crawling-indexing/consolidate-duplicate-urls) for SEO purposes

```suggestion
    <link rel="canonical" href="https://argo-workflows.readthedocs.io/en/stable/CONTRIBUTING/" />
    <meta http-equiv="refresh" content="0;URL=https://argo-workflows.readthedocs.io/en/stable/CONTRIBUTING/'" />
```

Reply:
latest == main, that's what will be generated when main merges (and this PR).

stable is determined by us on which SHA is tagged as `stable` which is not one of our current conventions.  I'd prefer `latest` and then semver tags.  wdyt?

Reply:
re: `stable` vs. `latest`, I see this was discussed [above](https://github.com/argoproj/argo-workflows/pull/12362#issuecomment-1860718112). True that `latest` is the equivalent to what we currently have, though `stable` is probably what would make more sense to use if we had that option (part of the reason for versioned docs is for unreleased future versions as I mentioned in https://github.com/argoproj/argo-workflows/issues/11390#issuecomment-1649834497). Nbd either way though I suppose.

> Actually, I guess the redirect takes one to "stable" - so that implies that if someone navigated to one of these files within a given Github release tag, they would not be redirected to readthedocs for that version, right?

@juliev0 there was no versioning for the docs previously, so there is no release tag for the docs / `gh-pages` branch in general. That is the main motivation for the move to Read the Docs, is to have versioned documentation.
The `gh-pages` branch is the rendered docs; they could still go to release tags of the plain markdown that's in source code's `docs/` branch

Reply:
👍 re: canonical

Reply:
> stable is determined by us on which SHA is tagged as `stable` which is not one of our current conventions. I'd prefer `latest` and then semver tags. wdyt?

the convention for `stable` is generally latest `major.minor.patch` (with some decisions about including RCs/beta/alpha), whereas `latest` includes unreleased features, which is generally not useful to a user as they generally can't use them (unless they use `latest` for their images).